webplugin/js/app/mck-app.js

```diff
@@ -468,6 +468,10 @@ function ApplozicSidebox() {
                 options.voiceNote != null
                     ? options.voiceNote
                     : widgetSettings && widgetSettings.voiceNote;
+            options.timeFormat24Hours =
+                options.timeFormat24Hours != null
+                    ? options.timeFormat24Hours
+                    : widgetSettings;          
             KommunicateUtils.deleteDataFromKmSession('settings');
```

Comment:
what if `options.timeFormat24Hours` is null?

Reply:
done